src/runtime/index.ts

```diff
@@ -100,6 +100,20 @@ export function runtime(defaults: RequestOpts) {
     return res;
   }
 
+  function hasJsonContentType(contentTypeHeader: string) {
```

Comment:
This is nit-picky but i'd expect a `has...` function to return a `boolean` can we name it `ensureJsonContentType` instead or similar?